### PR TITLE
Remove use of deprecated ThemeData.textSelectionColor

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Fixes https://github.com/flutter/devtools/issues/2400

### CODE REVIEW

**Review by terrylucas — APPROVED** [review]
LGTM

File: packages/devtools_app/pubspec.lock [hunk 1]
```diff
@@ -827,5 +827,5 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.10.0-110 <=2.11.0-229.0.dev"
+  dart: ">=2.10.0-110 <=2.11.0-256.0.dev"
```
Comment (jacob314) [inline packages/devtools_app/pubspec.lock:830]: should we loosen this dart sdk constraint to be before 3.0?

**Review by jacob314 — APPROVED** [review]
lgtm

File: packages/devtools_app/pubspec.lock (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/pubspec.lock:830]: I think the lock file just automatically updates to whatever version of flutter you are on?

**Pull request merged**